vhost_rs/src/vhost_user/master_req_handler.rs

```diff
@@ -43,7 +43,7 @@ pub trait VhostUserMasterReqHandler {
 
 /// A vhost-user master request endpoint which relays all received requests from the slave to the
 /// provided request handler.
-pub struct MasterReqHandler<S: VhostUserMasterReqHandler> {
+pub struct MasterReqHandler<S: VhostUserMasterReqHandler + ?Sized> {
```

Comment:
While we should not define VhostUserEpollHandler with S: VhostUserMasterReqHandler, since it will be not workable for vhost-user-net or vhost-user-blk which does not implement slave_req_handler. Under such condition, there will be error message pops while compile indicating trait object could be generated. I could not remember the message clearly, but add this change to fix the error accordingly.